Common/Data/Custom/Quiver/QuiverWallStreetBets.cs

```diff
@@ -0,0 +1,139 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using QuantConnect.Util;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using System;
+using System.IO;
+using NodaTime;
+using ProtoBuf;
+using static QuantConnect.StringExtensions;
+
+namespace QuantConnect.Data.Custom.Quiver
+{
+    /// <summary>
+    /// Mentions of the given company's ticker in the WallStreetBets daily discussion thread
+    /// </summary>
+    [ProtoContract(SkipConstructor = true)]
+    public class QuiverWallStreetBets : BaseData
+    {
+
+        /// <summary>
+        /// Date of the daily discussion thread
+        /// </summary>
+        [ProtoMember(10)]
+        [JsonProperty(PropertyName = "Date")]
+        [JsonConverter(typeof(DateTimeJsonConverter), "yyyy-MM-dd")]
+        public DateTime Date { get; set; }
+
+
+
+        /// <summary>
+        /// The follower count on the given date
```

Comment:
I think this comment is outdated, as well as a few more. I recommend reviewing the rest of the comments to make sure we have the right docs for these values.